examples/docs/src/pages/index.astro

```diff
@@ -0,0 +1,22 @@
+---
+import Layout from '../layouts/MainLayout.astro';
+---
+
+<script>
+  // WIP: trigger a client-side redirect based on the browser language.
+  // A vercel.json redirect is enforced in production, so no user should ever see this page.
+  // Remove the vercel.json redirect when this is ready.
+  const KNOWN_LANGUAGES = ['bg', 'de','en','es','fi','nl','pt-br','zh-CN','zh-TW', 'fr'];
```

Comment:
this example doesn't support all of these languages, so this would create a 404 for most of the languages listed here. Is there some way to auto-detect this instead? What about: 

```
// Note: this is pseudocode, doesn't actually run
const KNOWN_LANGUAGES = fs.scanDirSync('src/pages/').filter(item => item.isDirectory());
```

Reply:
Or, `Object.keys(SIDEBAR)` would also work!

Reply:
I'll see what I can do about this, but I'm not sure about language regions, e.g. en-us, en-uk, etc...

Reply:
the code handles all of that automatically, it just need to be a list of which directories we support.

Reply:
How do I send values from Astro to scripts? I get the dir. but I'm not sure how to send it to the actual script tag

Reply:
oh right, we're working on this feature now. In the meantime, the workaround is outlined in my comment here: #1107

Reply:
Ok